axon-framework/upgrading-to-4-7.md

```diff
@@ -121,3 +124,32 @@ default ones:
 | axon-eventsourcing-jakarta | axon-eventsourcing |
 | axon-messaging-jakarta     | axon-messaging     |
 | axon-modelling-jakarta     | axon-modelling     |
+
+## Migration Automation
+
+The steps above explain in detail what you need to do to upgrade to Axon Framework 4.7.
+If you want to automate some of these steps, there are two [OpenRewrite](https://github.com/openrewrite) migration recipes you can use:
+
+1. **Upgrade to Axon Framework 4.7 Jakarta** - A recipe to upgrade from an Axon Framework Javax-specific project to Jakarta.
```

Comment:
I think linking to the recipe on GitHub would be a helpful addition.

Reply:
Completely agree with you here. Added links!